arm_kinematics/CertesianPose startPose
arm_kinematics/CertesianPose endPose
---
bool feasible
